src/app/loadout/item-utils.ts

```diff
@@ -0,0 +1,13 @@
+import { DimItem } from 'app/inventory/item-types';
+import { armorStatHashes } from './known-values';
+
+/** Checks if the item is Armor 2.0 and whether it has stats present for all 6 armor stats. */
+export function isArmor2WithStats(item: DimItem) {
```

Comment:
This is isLoadoutBuilderItem but I have renamed and replaced statValues with armorStatHashes (they are both arrays of the 6 armor stat hashes.)